frontend/src/styled-theme.ts

```diff
@@ -26,7 +26,7 @@ export const breakpoints = {
   sm: '640px',
   md: '768px',
   xmd: '992px',
-  lg: '1024px',
+  lg: '1023px',
```

Comment:
I think we should leave this and target the root issue.  `1024` is a common breakpoint

Reply:
Will look into this further today.

Reply:
👍